### PR TITLE
[SPARK-31307][ml][EXAMPLES]Add examples for ml.fvalue

### PR DESCRIPTION
**Pull request by kevinyu98**
### What changes were proposed in this pull request?

Add FValue example for ml.stat.FValueTest in python/java/scala

### Why are the changes needed?

Improve ML example

### Does this PR introduce any user-facing change?

No
### How was this patch tested?

manually run the example

### CODE REVIEW

**Comment by kevinyu98** [comment]
@huaxingao

File: examples/src/main/java/org/apache/spark/examples/ml/JavaFValueTestExample.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import org.apache.spark.sql.types.Metadata;
+import org.apache.spark.sql.types.StructField;
+import org.apache.spark.sql.types.StructType;
+
+import java.util.Arrays;
+import java.util.List;
+
+// $example on$
+// $example off$
+
```
Comment (huaxingao) [inline examples/src/main/java/org/apache/spark/examples/ml/JavaFValueTestExample.java:37]: Let's rearrange the import like the following: 
```
import org.apache.spark.sql.SparkSession;

// $example on$
import java.util.Arrays;
import java.util.List;

import org.apache.spark.ml.linalg.Vectors;
import org.apache.spark.ml.linalg.VectorUDT;
import org.apache.spark.ml.stat.FValueTest;
import org.apache.spark.sql.Dataset;
import org.apache.spark.sql.Row;
import org.apache.spark.sql.RowFactory;
import org.apache.spark.sql.types.*;
// $example off$
```

Comment (kevinyu98) [reply examples/src/main/java/org/apache/spark/examples/ml/JavaFValueTestExample.java:34]: done, thanks.

**Comment by huaxingao** [comment]
LGTM 
cc @srowen

**Review by huaxingao — APPROVED** [review]

**Comment by srowen** [comment]
This goes into 3.0 right?

**Comment by huaxingao** [comment]
@srowen This is for 3.1 only.

**Comment by srowen** [comment]
Jenkins test this please

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**